sbnobj/Common/CRT/CRTHit.hh

```diff
@@ -22,17 +22,17 @@ namespace sbn::crt {
 
     struct CRTHit{
 
-      std::vector<uint8_t> feb_id; ///< FEB address 
+      std::vector<uint8_t> feb_id; ///< FEB address
       std::map< uint8_t, std::vector<std::pair<int,float> > > pesmap; ///< Saves signal hit information (FEB, local-channel and PE) .
       float         peshit; ///< Total photo-electron (PE) in a crt hit.
-     
+
       uint64_t       ts0_s; ///< Second-only part of timestamp T0.
-      int8_t    ts0_s_corr; ///< [Honestly, not sure at this point, it was there since long time (BB)]
-      
-      uint64_t      ts0_ns; ///< Timestamp T0 (from White Rabbit), in UTC absolute time scale in nanoseconds from the Epoch.
-      int8_t   ts0_ns_corr; ///< [Honestly, not sure at this point, it was there since long time (BB)]
-      int64_t       ts1_ns; ///< Timestamp T1 ([signal time w.r.t. Trigger time]), in UTC absolute time scale in nanoseconds from the Epoch.
-     
+      double    ts0_s_corr; ///< [Honestly, not sure at this point, it was there since long time (BB)]
```

Comment:
Since you are working on this, could someone actually clarify what is ts0_s_corr and ts1_ns_corr? Changing the data format is always difficult, so in case further changes are needed, it's more efficient to include them all at once.

Reply:
I tried to ask around in the past days but could find clarification on `ts0_s_corr` and `ts1_ns_corr`.